packages/cli/src/UserManagement/routes/owner.ts

```diff
@@ -79,18 +79,36 @@ export function ownerNamespace(this: N8nApp): void {
 
 			Logger.info('Owner updated successfully', { userId: req.user.id });
 
-			config.set('userManagement.hasOwner', true);
-
 			await Db.collections.Settings!.update(
 				{ key: 'userManagement.hasOwner' },
 				{ value: JSON.stringify(true) },
 			);
 
+			config.set('userManagement.hasOwner', true);
+
 			Logger.debug('Setting hasOwner updated successfully', { userId: req.user.id });
 
 			await issueCookie(res, owner);
 
 			return sanitizeUser(owner);
 		}),
 	);
+
+	/**
+	 * Persist that the instance owner setup has been skipped
+	 */
+	this.app.post(
+		`/${this.restEndpoint}/owner/skip-setup`,
+		// eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
This is a common use case, so you could disable the default selector:
https://github.com/n8n-io/n8n/blob/401e626a6429fc45d84dcede756b69d8a6c55e75/.eslintrc.js#L213-L216